src/main/java/seedu/address/commons/core/increment/Increment.java

```diff
@@ -3,14 +3,18 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
+import seedu.address.model.person.Salary;
+import seedu.address.model.person.SalaryParserUtil;
+
 /**
  * Represents an increment.
  */
 public class Increment {
```

Comment:
I think ideally, the `Increment` constructor should have taken in a `long` argument already, since the IncrementCommandParser should be parsing the raw string input to send the required data type to the Command.

But for now we can probably merge this first so clear the bugs, then improve code quality if we still have time

Reply:
Yup, that's a good point! I'm not sure what error message to show to the users though, because the increment needs to be below max long / 100, but implementing a maximum increment feels like a feature change and I can't think of another way to implement this

Reply:
That's true, it's probably not possible to introduce a new error message at the parser now. It should be fine!